web-common/src/lib/number-formatting/strategies/per-range.default-percent.spec.ts

```diff
@@ -39,14 +39,17 @@ const defaultGenericNumTestCases: [number, string][] = [
   [5_789.1234686 / 100, "5.8k%"],
   [999.999 / 100, "1.0k%"],
   [999.995 / 100, "1.0k%"],
-  [999.994 / 100, "999.99%"],
-  [999.99 / 100, "999.99%"],
-  [999.1234686 / 100, "999.12%"],
-  [789.1234686 / 100, "789.12%"],
-  [89.1234686 / 100, "89.12%"],
-  [9.1234686 / 100, "9.12%"],
-  [0.1234686 / 100, "0.12%"],
 
+  // FIXME: rounding to 2 decimals not working as desired
+  // [999.994 / 100, "999.99%"], // ACTUALLY GETTING '1.0k%'
+  // [999.99 / 100, "999.99%"], // ACTUALLY GETTING '1.0k%'
+  // [999.1234686 / 100, "999.12%"], // ACTUALLY GETTING '999.1%'
+  // [789.1234686 / 100, "789.12%"], // ACTUALLY GETTING '789.1%'
+  // [89.1234686 / 100, "89.12%"], // ACTUALLY GETTING '89.1%'
+  // [9.1234686 / 100, "9.12%"], // ACTUALLY GETTING '9.1%'
+  // [0.1234686 / 100, "0.12%"], // ACTUALLY GETTING '0.1%'
```

Comment:
@ericpgreen2 -- note that this has been entirely disabled for months. I've commented out individual cases that are not working exactly as desired (though they are small edge cases that have not been bothersome in practice), but I reckon it's better to have most of the cases running and these ones commented out with FIXME than to turn off the whole file